runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7524,6 +7524,13 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
       vmThread
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
    );
+   if(TR::Options::getDebug() && !cannotDoRemoteCompilation  && _compInfo.getPersistentInfo()->getRemoteCompilationMode() == JITServer::CLIENT)
```

Comment:
One last thing: to follow our coding conventions, please add a space between `if` and `(`